Makefile

```diff
@@ -8,7 +8,7 @@ SDSL_DIR=../sdsl-lite
 #VERIFY_FLAGS=-DVERIFY_CONSTRUCTION
 
 # Multithreading with OpenMP and libstdc++ Parallel Mode.
-PARALLEL_FLAGS=-fopenmp -pthread -D_GLIBCXX_PARALLEL
+PARALLEL_FLAGS=-fopenmp -pthread
```

Comment:
Is it possible to remove the flag for clang builds, until gcsa2 can migrate from libstdc++?